datafusion/core/tests/sql/order.rs

```diff
@@ -198,3 +199,25 @@ async fn sort_empty() -> Result<()> {
     assert_eq!(results.len(), 0);
     Ok(())
 }
+
+#[tokio::test]
+async fn sort_with_lots_of_repetition_values() -> Result<()> {
```

Comment:
❤️  love the tests

Reply:
BTW I ran the test locally without the changes in this PR to confirm coverage:

```shell
cargo test -p datafusion --test sql_integration -- sort_with_lots_of_repetition_values
```

They failed with:

```

---- sql::order::sort_with_lots_of_repetition_values stdout ----
thread 'sql::order::sort_with_lots_of_repetition_values' panicked at 'assertion failed: `(left == right)`
  left: `Some(2451809)`,
 right: `Some(2451816)`', datafusion/core/tests/sql/order.rs:220:9
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

👍